build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -x
+
+if [[ "$1" == "" ]]; then
+    BUILD_DIR="${HOME}/.local/share/coredns-machine-kubernetes-build"
+    if [[ ! -d "${PWD}/build" ]]; then
+        mkdir -p "$BUILD_DIR"
+    fi
+    mkdir -p "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes"
+    cp -r . "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes"
+    chcon -Rv -t container_file_t "$BUILD_DIR"
+    sudo podman run --rm \
```

Comment:
This still fails for me with a DNS error. Not really sure why.

Reply:
Which step fails? sudo podman run?